test/__snapshots__/loader.test.js.snap

```diff
@@ -733,6 +733,27 @@ Array [
     "./index-loader-syntax.css",
     ".a {
   color: red;
+}",
+    "",
+  ],
+  Array [
+    "button.modules.css!=!./index-loader-syntax-sass.css",
+    ".C6WgXrKWbFsSifATWfzu {
+  width: 5px;
+}",
+    "",
+  ],
+  Array [
+    "button.module.scss!=!./base64-loader/index.js?[ident]!./simple.js?foo=bar",
+    "._1Z1CpM4SYZX9egsUO4gNTz {
+  color: red;
+}",
+    "",
+  ],
+  Array [
+    "other.module.scss!=!./base64-loader/index.js?[ident]!./simple.js?foo=baz",
+    "._1KqQu-H04lA8NLg4-NKnDf {
+  color: red;
```

Comment:
You can see here: hash for classes are different

Reply:
But note: you should use different `matchResource`, otherwise hash will be same

Reply:
perfect!